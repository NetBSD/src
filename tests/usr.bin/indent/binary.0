/*	$NetBSD: binary.0,v 1.1.2.2 2019/06/10 22:10:12 christos Exp $	*/
/* $FreeBSD: head/usr.bin/indent/tests/binary.0 318471 2017-05-18 17:15:58Z pstef $ */
#define b00101010 -1
void t(void) {
	unsigned a[] = {0b00101010, 0x00005678, 02, 17U};
	float x[] = {.7f, 0.7f};
	unsigned long ul[] = {0b00001111UL, 0x01010101UL, 02UL, 17UL};

	if (0 b00101010)
		return;
}
