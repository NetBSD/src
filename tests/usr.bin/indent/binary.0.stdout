/*	$NetBSD: binary.0.stdout,v 1.1.2.2 2019/06/10 22:10:12 christos Exp $	*/
/* $FreeBSD: head/usr.bin/indent/tests/binary.0.stdout 321381 2017-07-23 14:04:45Z pstef $ */
#define b00101010 -1
void
t(void)
{
	unsigned	a[] = {0b00101010, 0x00005678, 02, 17U};
	float		x[] = {.7f, 0.7f};
	unsigned long	ul[] = {0b00001111UL, 0x01010101UL, 02UL, 17UL};

	if (0 b00101010)
		return;
}
